src/site/markdown/server/manage-webadmin.md

```diff
@@ -0,0 +1,334 @@
+Web administration for JAMES
+============================
+
+The web administration supports for now the CRUD operations on the domains, the users and the users mailboxes, and quotas, as described in the following sections.
+
+**WARNING**: This API allow authentication only via the use of JWT. It means that, otherwise, an administrator should ensure an attacker can not use this API.
+
+Please also note **webadmin** is only enabled with **Guice**. You can not use it when using James with **Spring**.
```

Comment:
Because we nether bothered enabling it with Spring? Because don't like it, and don't want to invest time in it?